### ISSUE TITLE
Timer not starting automatically since Chrome 49

### ISSUE THREAD

**Issue by Lilchef** [Lilchef/jira-time-logger-chrome#8]
Newer versions of Chrome (since 49), seems to have caused the stopwatch to break and stick at 0m.
Manually resetting the time starts it up and manually entering time and logging it works fine, although I've seen in the inspector that when I do so we get:
"js/Stopwatch.js:296 Uncaught TypeError: Cannot read property 'getTime' of null"
This is on the result of getStart() so the start time has not been initialised for some reason.

**Comment by Lilchef** [comment]
I've narrowed this down to the 'config initialised' message being sent ([Config.js L174](https://github.com/Lilchef/jira-time-logger-chrome/blob/master/src/js/Config.js#L174)) but not being received on the other end ([index.js L35](https://github.com/Lilchef/jira-time-logger-chrome/blob/master/src/js/index.js#L35)) so the app never initialises and, therefore, doesn't start the timer.
Apparently, this only used to work prior to Chrome 49 due to a 'bug' that got fixed ([link](http://stackoverflow.com/a/35858654))

**Comment by Lilchef** [comment]
That change in Chrome meant that the background page could no longer send messages to itself, only to the UI page. To address that I had to use traditional events. However, the background page *cannot* talk to the UI page via events, only messages, so we need both.

I made the event layer always create both a message and an event which worked but, frustratingly, the UI page picks up on both (even though the background page doesn't) so it was doing everything twice.
The easiest thing to do was just prevent things from happening twice.
